client/src/locales/en.json

```diff
@@ -85,8 +85,8 @@
   "about.intro-heading": "{appName} Office inventory tracking app",
   "about.intro-subheading": "{appName} helps you track all the office inventory you need!",
   "user.anonymous": "Anonymous",
-  "user.account-deleted": "Your account has been successfully deleted.",
-  "user.account-deleted-message": "Your account has been successfully deleted. Click OK to continue.",
+  "user.account-deleted": "Your account has been successfully deleted. ",
```

Comment:
White space at the end of the message. This message will be joined with another message so space is needed there will always be this space even if needed.
I'm not sure if this is correct but maybe try add space in component:
```
<FormatMessage />
{' '}
<FormatMessage />
```